relay/Cargo.toml

```diff
@@ -26,11 +26,8 @@ hostname = "0.3.1"
 lazy_static = "1.4.0"
 relay-common = { path = "../relay-common" }
 relay-config = { path = "../relay-config" }
-relay-log = { path = "../relay-log" }
+relay-log = { path = "../relay-log", features = ["init"] }
```

Comment:
This adds the full `sentry` crate to the dependency tree, which includes a dependency on `openssl`. When compiling `relay-cabi`, this feature is disabled.